pyproject.toml

```diff
@@ -291,6 +291,13 @@ dl = [
   'transformers[torch]<4.41.0; python_version < "3.12"',
   'pykan<0.2.2,>=0.2; python_version > "3.9.7"',
   'pytorch-forecasting>=1.0.0; python_version < "3.11"',
+  'lightning>=2.0; python_version < "3.12"',
+  'hydra-core>=1.3; python_version < "3.12"',
+  'gluonts>=0.14.3; python_version < "3.12"',
+  'einops>0.7.0; python_version < "3.12"',
+  'jax[cpu]; python_version < "3.12"',
+  'jaxtyping>=0.2.24; python_version < "3.12"',
```

Comment:
I think this can be removed everywhere by removing the usage of the corresponding types. This might help to remove Jax itself. Please check it :)